OOP-1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="./movies.js"></script>
+    <script type="text/javascript">
+    maxi = [];
+    ln=[];
+    let x;
+    let y;
+
+    class MaxCast {
```

Comment:
This class need not be restricted to finding the movie with the maximum number of cast. It could have other methods too. Classes can be as generic as possible. They essentially represent a blueprint of an object, you can think about what object this class represents.